releases/SN30_Pro+/3.0195/sn30pro+.metainfo.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2016 Richard Hughes <richard@hughsie.com> -->
+<!-- Copyright 2019 Mario Limonciello <mario.limonciello@dell.com> -->
+<component type="firmware">
+  <id>com.8bitdo.sn30pro+.firmware</id>
+  <name>SN30 Pro+</name>
+  <summary>Firmware for the 8Bitdo SN30 Pro+ Game Controller</summary>
+  <description>
+    <p>
+      Updating the firmware on your SN30 Pro+ device improves performance and adds
+      new features.
+    </p>
+  </description>
+  <provides>
+    <!-- USB\VID_2DC8&PID_5750&REV_020 -->
+    <firmware type="flashed">78acf201-a624-5c0b-a521-e204be14e3d5</firmware>
+    <!-- USB\VID_2DC8&PID_5750 -->
+    <firmware type="flashed">b02f1319-c1c4-54f4-b7f2-5bb3552f6a9e</firmware>
+  </provides>
+  <url type="homepage">http://www.8bitdo.com/sn30pro-plus/</url>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>Proprietary</project_license>
+  <developer_name>8Bitdo</developer_name>
+  <releases>
+    <release urgency="medium" version="3.0195" date="2019-11-17" timestamp="1574059883">
```

Comment:
That's not possible....

	tmp = g_strdup_printf ("%u.%02u", version / 100, version % 100);
	fu_device_set_version (FU_DEVICE (self), tmp, FWUPD_VERSION_FORMAT_PAIR);

the second number has to be < 100...

Reply:
Well that's unfortunate.  We don't really have a good way to represent a beta release then.